Libraries/Components/View/ReactNativeStyleAttributes.windows.js

```diff
@@ -0,0 +1,48 @@
+/**
+ * Copyright (c) 2015-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @providesModule ReactNativeStyleAttributes
```

Comment:
>ReactNativeStyleAttributes [](start = 19, length = 26)

What's the difference between ViewStylePropTypes and ReactNativeStyleAttributes?

Reply:
I think the 1st defines all the properties and styles while the 2nd tells which properties are colors and injects a parser for values like `rgba(1, 2, 3, 4)` so the C# RN code would always get 32-bit values.